ivy/functional/backends/paddle/statistical.py

```diff
@@ -89,8 +114,21 @@ def max(
     return ret.astype(ret_dtype)
 
 
-@with_supported_dtypes(
```

Comment:
There's a current problem with `with_supported_dtypes`, but that should be fixed soon. I don't think we should stop using `with_supported_dtypes` or `with_unsupported_dtypes` in favour of `with_supported_device_and_dtypes` and `with_unsupported_device_and_dtypes` as both types of decorators are supposed to do the same thing based on convenience 😄

Reply:
> There's a current problem with `with_supported_dtypes`, but that should be fixed soon. I don't think we should stop using `with_supported_dtypes` or `with_unsupported_dtypes` in favour of `with_supported_device_and_dtypes` and `with_unsupported_device_and_dtypes` as both types of decorators are supposed to do the same thing based on convenience 😄

@vedpatwardhan Okay got it. For the time being I have shifted to these decorators as there are functions failing even after correcting the decorators. For example mean gives assertion error as backend values and grouth truth values dosent match. Will look into each function and notify you accordingly!

Reply:
this issue has been fixed on `main` now, we can revert back to using `with_supported_dtypes` 😄